discovery-infra/test_infra/tools/terraform_utils.py

```diff
@@ -14,14 +15,19 @@ def __init__(self, working_dir):
         self.init_tf()
 
     def init_tf(self):
-        self.tf.cmd("init -plugin-dir=/root/.terraform.d/plugins/", raise_on_error=True)
+        try:
+            self.tf.cmd("init -plugin-dir=/root/.terraform.d/plugins/", raise_on_error=True)
+        except:
+            logging.exception("Init")
+            time.sleep(30000)
```

Comment:
How would sleep help?

Reply:
@ori-amizur ^^

Reply:
@empovit Please remove.  It was for debug only

Reply:
Done